jdaviz/tests/test_show.py

```diff
@@ -1,17 +1,32 @@
 import sidecar
+from pytest import warns
 
 
 def test_show_inline(specviz_helper):
-    res = specviz_helper.show_inline()
-
+    res = specviz_helper.show(mode='inline')
     assert res is None
 
 
 def test_show_sidecar(specviz_helper):
-    res = specviz_helper.show_in_sidecar()
+    res = specviz_helper.show(mode='sidecar')
     assert isinstance(res, sidecar.Sidecar)
 
 
-def test_show_new_tab(specviz_helper):
-    res = specviz_helper.show_in_new_tab()
+def test_show_new_jupyter_tab(specviz_helper):
+    res = specviz_helper.show(mode="new jupyter tab")
     assert isinstance(res, sidecar.Sidecar)
+
+
+def test_show_new_browser_tab(specviz_helper):
+    with warns(RuntimeWarning):
```

Comment:
If you want to be more explicit, you can also try to match the warning message as follows. It also takes regex if you wish.
```suggestion
    with warns(RuntimeWarning, match='Error detected in display'):
```